packages/mobile/src/page/Cafeteria/index.tsx

```diff
@@ -24,13 +23,17 @@ function Cafeteria() {
     date: dayjs().date(),
     day: dayjs().day(),
   });
+
   const dispatch = useAppDispatch();
   const { selectedMenu } = useAppSelector((state) => {
     return state.persistedReducer.cafeteria.cafeteria;
   });
   const handleMenu = (id: number) => {
     dispatch(selectMenu({ selectedMenu: id }));
   };
+  const allCafeteriaData = useCafeteria(`${year}-${month + 1}-${date}`);
+  const { isLoading, data, isError } = allCafeteriaData[selectedMenu - 1];
+  const cafeteriaMenu: res.Cafeteria[] | undefined = data?.data;
```

Comment:
이 부분 타입 선언 없어도 작성하신 타입과 똑같이 추론되지 않나용?

```suggestion
  const cafeteriaMenu = data?.data;
```

Reply:
마자용 추론되여 지울게요~